examples/wifi/expanded/requests_wifi_multiple_cookies.py

```diff
@@ -1,36 +1,38 @@
 # SPDX-FileCopyrightText: 2022 Alec Delaney
 # SPDX-License-Identifier: MIT
-
-"""
-This example was written for the MagTag; changes may be needed
-for connecting to the internet depending on your device.
-"""
+# Coded for Circuit Python 9.0
+""" Multiple Cookies Example written for MagTag """
+# pylint: disable=import-error
```

Comment:
I don't think this pylint disable is required is it? I checked out this branch locally and removed this line then ran `pre-commit run -a` and it succeeded without any pylint errors.

Reply:
@FoamyGuy This is good to go too. Lots of PR's to check up on.